shared/utils/queue.go

```
@@ -1,46 +1,272 @@
+func (hp HadesProducer) EnqueueLowJob(ctx context.Context, queuePayloud payload.QueuePayload) error {
+	return hp.EnqueueJobWithPriority(ctx, queuePayloud, LowPriority)
+}
+
+func (hp HadesProducer) EnqueueJobWithPriority(ctx context.Context, queuePayloud payload.QueuePayload, priority Priority) error {
+	bytesPayload, err := json.Marshal(queuePayloud)
+	if err != nil {
+		slog.Error("Failed to marshal payload", "error", err)
+	}
+	_, err = hp.js.PublishAsync(PrioritySubject(priority), queuePayloud.ID[:], jetstream.WithMsgID(queuePayloud.ID.String()))
+
+	hp.kv.Put(ctx, queuePayloud.ID.String(), bytesPayload)
```

golangci-lint (1.64.8)

Error return value of `hp.kv.Put` is not checked

(errcheck)